specification/domainservices/resource-manager/Microsoft.AAD/2017-06-01/domainservices.json

```diff
@@ -0,0 +1,586 @@
+            }
+          },
+          "204": {
+            "description": "HTTP 204 (Not Content) should be used if the resource does not exist and the request is well formed."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Delete Domain Service": {
+            "$ref": "./examples/DeleteDomainService.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "DomainServices"
+        ],
+        "operationId": "DomainServices_Update",
+        "summary": "Update Domain Service (PATCH Resource)",
+        "description": "The Update Domain Service operation can be used to update the existing deployment. The update call only supports the properties listed in the PATCH body.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "domainServiceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the domain service in the specified subscription and resource group."
+          },
+          {
+            "name": "properties",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DomainServicePatchProperties"
+            },
+            "description": "Properties supplied to the Update a Domain Service operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "HTTP 200 (OK) if the operation was successful.",
+            "schema": {
+              "$ref": "#/definitions/DomainService"
+            }
+          },
+          "202": {
+            "description": "HTTP 202 (Accepted) if the operation was successfully started and will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DomainService"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Update Domain Service": {
+            "$ref": "./examples/UpdateDomainService.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Resource": {
+      "description": "The Resource model definition.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location"
```

Comment:
It would be nice to have this line `"x-ms-mutability": ["create", "read"]` for the `location` property. It's only for SDK generation to say that the `location` is a read-only property, which makes because it's impossible to change location of Azure resource anyway.